tests/test_logging.py

```diff
@@ -19,20 +19,23 @@ def test_default_handler(capsys):
 
     # We need to reconstruct our default handler to properly capture stderr.
     optuna.logging._reset_library_root_logger()
+    optuna.logging.set_verbosity(optuna.logging.INFO)
+
     library_root_logger = optuna.logging._get_library_root_logger()
+    library_root_logger.propagate = False
     example_logger = optuna.logging.get_logger('optuna.bar')
 
     # Default handler enabled
     optuna.logging.enable_default_handler()
     assert library_root_logger.handlers
-    example_logger.warning('hey')
+    example_logger.info('hey')
     _, err = capsys.readouterr()
     assert 'hey' in err
 
     # Default handler disabled
     optuna.logging.disable_default_handler()
     assert not library_root_logger.handlers
-    example_logger.warning('yoyo')
```

Comment:
I don't know the reason, but when `example_logger.warning('yoyo')` is called, the plain text `yoyo` is outputted to the standard error even if Optuna's log handler has been disabled.

Reply:
It's Python's default handler:

https://docs.python.org/3/howto/logging.html#advanced-logging-tutorial
> If you call the functions debug(), info(), warning(), error() and critical(), they will check to see if no destination is set; and if one is not set, they will set a destination of the console (sys.stderr) and a default format for the displayed message before delegating to the root logger to do the actual message output.

Reply:
I see. Thank you for your information!